conda/pytorch-nightly/meta.yaml

```diff
@@ -23,13 +23,12 @@ requirements:
     - mkl-include # [x86_64]
     - mkl=2020.2 # [py <= 311 and x86_64 and not win]
     - mkl=2023.1 # [py >= 312 and x86_64]
-    - mkl-devel=2021.4.0 # [x86_64 and win and py<=311]
-    - mkl-devel=2023.1 # [x86_64 and win and py>=312]
+    - mkl-devel=2023.1 # [x86_64 and win]
     {% endif %}
     - typing_extensions
     - ninja
     - libuv # [win]
-    - numpy=1.19 # [py == 38]
+    - numpy=1.22.3 # [py == 38]
     - numpy=2.0.0rc1 # [py >= 39]
```

Comment:
Not for this PR, now that 2.0 is fully released, we can update all of these to point to the official release and not the rc